### PR TITLE
Playlist UI improvements to match mockups

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
- Match [mockup styles](https://overflow.io/s/BFZSP3CC?node=d7a08ba5)
- Move Add button to top
- Icon for embedded templates
- Item name ellipsis
- Cleanup no longer used css classes

## Motivation and Context
Playlist UI epic

## How Has This Been Tested?
Locally and on [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/scss/sections/components/playlist.scss [hunk 1]
```diff
@@ -1,17 +1,19 @@
+.rise-playlist-container, .presentation-selector-container, .playlist-item-container {
```
Comment (alex-deaconu) [inline src/scss/sections/components/playlist.scss:1]: @ezequielc I'm not really sure if all these classes need these variables. I just added them in when splitting the pages but didn't double check. Any ideas? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, thanks for the updates!

File: src/scss/sections/components/playlist.scss (hunk 1 above)
Comment (ezequielc) [reply src/scss/sections/components/playlist.scss:1]: @alex-deaconu At least the first two are needed. I will check the other one and remove in a future PR if not needed. Thanks

Comment (alex-deaconu) [reply src/scss/sections/components/playlist.scss:1]: @ezequielc sounds good, thanks!

**Pull request merged**